src/webview/inspectorView.ts

```diff
@@ -6,13 +6,19 @@ const vscode = acquireVsCodeApi();
 
 window.addEventListener("load", main);
 
-window.addEventListener('click', (event) => {
-  const clickedElement = event.target
-  // Check if the clicked element is not inside the webview
-  if (!document.body.contains(clickedElement as Node)) {
-    vscode.postMessage({ type:'command', value: 'removeClass' });
+const focusedElement = document.querySelector('.focus')
+
+if (focusedElement) {
+  focusedElement.scrollIntoView()
+
+  function clickHandler() {
+    focusedElement?.classList.remove('focus')
+    window.removeEventListener('click', clickHandler)
   }
-});
+
+  window.addEventListener('click', clickHandler)
```

Comment:
you should also be able to pass the `once` parameter that will automatically remove the listener when it's invoked https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#parameters